docs/change-log/index.md

```diff
@@ -27,9 +27,11 @@
     2. Within the Jupyter service, open a terminal and run:
      
 	```
-	 conda create -n python39 python=3.9 ipykernel -y
-      conda activate python39
-      ./align_mlrun.sh
+    install -y conda=23.1.0
+    install -y pip
```

Comment:
it's best to keep the $CONDA_HOME/bin/conda prefix. it should solve inconsistencies the user may have with the packages
```$CONDA_HOME/bin/conda install -y conda=23.1.0
$CONDA_HOME/bin/conda install -y pip

Reply:
@alonmr why not `~=23.1.0` ?

Reply:
kept `$CONDA_HOME/bin/conda`